Source/Scene/Cesium3DTileset.js

```diff
@@ -788,19 +816,57 @@ define([
         }
     }
 
-    function getScreenSpaceError(geometricError, tile, frameState) {
+    var scratchPositionNormal = new Cartesian3();
+
+    function updateDynamicScreenSpaceError(tileset, frameState) {
+        var camera = frameState.camera;
+        var positionCartographic = camera.positionCartographic;
+        var height = positionCartographic.height;
+
+        // The range where the density starts to lessen
```

Comment:
I added a part-way solution. It now uses the bounding volume's center as a starting point. However I wasn't sure how to get the "height" of the tileset in a consistent way. It's easy for regions but obb's would need to be aligned z-up, and for bounding spheres it's hard to know at all.